app/model.js

```diff
@@ -120,5 +120,52 @@ const fetchArticleById = (params) =>
 }
 
 
-module.exports = {fetchTopics, fetchArticles, fetchArticleById, fetchComments}
+
+const changeArticleVotes = (article_id, body) =>
+{
+    if(!/[0-9]+/g.test(article_id))
+    {
+        return Promise.reject({status: 400, msg: 'invalid article ID: not a number'})
+    }
+    if(!/[0-9]+/g.test(body.inc_votes))
+    {
+        return Promise.reject({status: 400, msg: 'invalid vote input: not a number'})
+    }
+    return db.query
+    (
+        `
+        UPDATE articles
+        SET votes = votes + $1
+        WHERE article_id = $2
+        `
+        ,
```

Comment:
It might be cleaner to pop a RETURNING* here rather than have a fresh query to the db in the next db